docs/UserGuide.md

```diff
@@ -524,8 +546,106 @@ To check that your system has the correct Java version (Java 11 and above) to ru
 
 5. If you do not see a similar result in the terminal after Step 3, or have an earlier version of Java, head over to [Oracle's Java download page](https://www.oracle.com/java/technologies/downloads/) to install Java.
 
+
+### 8.2 I cannot see any data
+
+Here are a few reasons why this may have occurred:
+
+1. The data file is empty:
+    * Locate the [date file](#6-faq).
+    * If the file is empty ,i.e. contains no data, then *MyInsuRec* did not load any data on start up.
+    * How to recover? :
+    * [Swap the file with your backup](#553-creating-and-using-data-file-backups).
+2. The data file was corrupt before starting the app:
+    * This issue is harder to detect since the app will only empty the file if the data in it was corrupt before starting.
+    * How to avoid this issue? :
+    * [Create backups](#553-creating-and-using-data-file-backups) of your `myinsurec.json` file and ensure that they are not corrupt.
+    * See the [section below](#83-my-data-file-is-corrupt) on how to detect and recover from file corruption.
+
+### 8.3 My data file is corrupt
+
+Do not launch the app if you detect this issue, since the `myinsurec.json` file will be cleared of all data.
```

Comment:
How can they detect this issue?

Reply:
Hmm, I'm not too sure how to explain this, would probably like to get feedback from the rest!